src/States/MainReportsUiState.cpp

```diff
@@ -206,7 +206,7 @@ void MainReportsUiState::initialize()
 	Panels[PANEL_SPACEPORT].Name = "Space Ports";
 
 	Renderer& r = Utility<Renderer>::get();
-	setPanelRects(r.width());
+	setPanelRects(static_cast<int>(std::floor(r.width())));
```

Comment:
Ahh, this caused the build to break on Clang since there is no `<cmath>` include for `std::floor`.

Reply:
Btw, why was `std::floor` added here?

Reply:
I haven't the foggiest idea why I added it.

Reply:
Fixed in #180.